### PR TITLE
V5 ga 23 divider

### PR DESCRIPTION
**Pull request by SrdjanReblaze**
https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?selectedIssue=V5GA-23

### CODE REVIEW

File: src/components/LayoutDividerLine.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<template>
+  <hr
+    class="divider-line"
```
Comment (Aviv-Galmidi) [inline src/components/LayoutDividerLine.vue:3]: Please keep the first attribute line in the same line as the HTML element

File: src/components/LayoutDividerLine.vue [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    :class="[color, orientation]"
+    :orientation="orientation"
+    :color="color"
+  />
```
Comment (Aviv-Galmidi) [inline src/components/LayoutDividerLine.vue:7]: Please keep the closing tag in the same line as the last attribute

File: src/components/LayoutDividerLine.vue [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+.divider-line {
+  margin: 5px 0;
+
+  &.gray {
+    border: 1px solid $color-dusty-gray;
+  }
+
+  &.light-gray {
+    border: 1px solid $color-silver;
+  }
```
Comment (Aviv-Galmidi) [inline src/components/LayoutDividerLine.vue:38]: Can we move the `1px solid` from both `&.gray` and `&.light-gray` to the basic `.divider-line`?
```suggestion
.divider-line {
  margin: 5px 0;
  border: 1px solid;

  &.gray {
    border-color: $color-dusty-gray;
  }

  &.light-gray {
    border-color: $color-silver;
  }
```

File: src/components/LayoutDividerLine.vue (hunk 1 above)
Comment (SrdjanReblaze) [reply src/components/LayoutDividerLine.vue]: updated

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**